resources/js/components/small/AdminShowMessage.vue

```diff
@@ -0,0 +1,19 @@
+<template>
```

Comment:
And then this one also to components/tabs/admin/components and renamed to ShowMessage

Reply:
done